app/models/spree/user.rb

```diff
@@ -35,7 +35,7 @@ def password_required?
 
       def set_login
         # for now force login to be same as email, eventually we will make this configurable, etc.
-        self.login ||= self.email if self.email
+        self.login = self.email
```

Comment:
@vinay-mittal  @krzysiek1507  But then what is the meaning of keeping extra parameter as login.

Its realy good option to have login field separate, It useful in mobile to have mobile number for login parameters.